core/src/consensus.rs

```diff
@@ -999,16 +1002,16 @@ impl Tower {
                         if old_vote.slot() == vote.slot()
                             && old_vote.confirmation_count() == vote.confirmation_count()
                         {
-                            return true;
+                            return None;
                         }
                     }
                 }
-                lockout > self.threshold_size
+                (lockout <= self.threshold_size).then_some(*fork_stake)
             } else {
-                false
+                Some(0)
```

Comment:
Might be good to clarify what is happening for this case w/ a comment. I believe this is the case where we haven't seen any votes on this fork so fork_stake hasn't yet been populated.